src/components/GamesCard/GamesCard.jsx

```diff
@@ -1,15 +1,21 @@
 import { useState } from "react";
 import { Link } from "react-router-dom";
-import { addToCartHandler } from "../../utilities";
-import { useCart, useAuth, useAuthModal } from "../../context";
+import {
+  addToCartHandler,
+  addToWishlistHandler,
+  removeFromWishlistHandler,
+} from "../../utilities";
+import { useCart, useAuth, useAuthModal, useWishlist } from "../../context";
 import { ICONS_URL } from "../../constants";
 import styles from "./GamesCard.module.css";
 
 const GamesCard = ({ item }) => {
   const { cartDispatch } = useCart();
   const { authState } = useAuth();
+  const { wishlistDispatch } = useWishlist();
   const { authModalHandler } = useAuthModal();
   const [itemAdded, setItemAdded] = useState(false);
+  const [isWishlisted, setIsWishlisted] = useState();
```

Comment:
Bug: When I click on heart icon in product listing page to add it to wishlist, the heart icon becomes solid. If i go to some other page and come back to product listing, the heart icon becomes outline again.

Reply:
This would get sorted once we have a proper backend that would have the data persisting. That time what I would do is compare the products array with the wishlist array and then make all those in the wishlist as `solid-heart-icon` as that is already added in the wishlist.